testkit/testkit.go

```diff
@@ -101,6 +102,17 @@ func (tk *TestKit) ResultSetToResultWithCtx(ctx context.Context, rs sqlexec.Reco
 	return &Result{rows: rows, comment: comment, assert: tk.assert, require: tk.require}
 }
 
+// HasPlan checks if the result execution plan contains specific plan.
+func (tk *TestKit) HasPlan(sql string, plan string, args ...interface{}) bool {
```

Comment:
Whether we use `testkit/testkit.go` as a substitute of `util/testkit/testkit.go`?

Reply:
Yes. `util/testkit` depends on `pingcap/check` which will be removed after #26022 done. To avoid patching too much on existing and to be deprecated code, also to reduce the size of `util`, I create the package and will later remove `util/testkit`.